core/src/com/unciv/logic/map/mapgenerator/MapLandmassGenerator.kt

```diff
@@ -141,10 +142,11 @@ class MapLandmassGenerator(val ruleset: Ruleset, val randomness: MapGenerationRa
         return min(0.3, 1.0 - (5.0 * distanceFactor * distanceFactor + randomScale) / 3.0)
     }
 
-    private fun getTwoContinentsTransform(tileInfo: TileInfo, tileMap: TileMap): Double {
+    private fun getTwoContinentsTransform(tileInfo: TileInfo, tileMap: TileMap, isLatitude: Boolean): Double {
```

Comment:
Rather than having this be random, it should be according to the map dimensions. If width > height split along the middle width, if height > width split along the middle height

Reply:
So I coded this, then went to test it, and I can't, because you can only define the map radius not width and height.

Reply:
Oh, you can on rectangular just not on hexagonal. I see.

Reply:
Done!

Reply:
It is still random when the width and height are the same.